packages/tools/client-debugger/client-debugger/src/Registry.ts

```diff
@@ -73,6 +74,19 @@ export interface FluidClientDebuggerProps {
 	 * debugger instances.
 	 */
 	containerNickname?: string;
+
+	/**
+	 * (optional) Configurations for generating visual representations of
+	 * {@link @fluidframework/shared-object-base#ISharedObject}s under {@link FluidClientDebuggerProps.containerData}.
+	 *
+	 * @remarks
+	 *
+	 * If not specified, then only `SharedObject` types natively known by the system will be visualized, and using
+	 * default visualization implementations.
+	 *
+	 * Provided visualizer configurations will be used in place of system defaults if specified.
```

Comment:
```suggestion
	 * If a visualizer configuration is specified for a shared object type that has a default visualizer, the custom one will be used.
```
To make it clear that the object with custom visualizers doesn't blindly replace the defaults object; it adds/overrides on a per-key basis.